silverback/application.py

```diff
@@ -59,22 +75,50 @@ def __init__(self, settings: Optional[Settings] = None):
 
     def on_startup(self) -> Callable:
         """
-        Code to execute on startup / restart after an error.
+        Code to execute on worker startup / restart after an error.
+
+        Usages example::
```

Comment:
`Usages` sounds funny to me, should this be `Usage` everywhere? I thought it was a typo at first

Reply:
I thought it was some sort of standard in google docstrings lol

Reply:
i think the two `::` means it is a custom section

Reply:
Ahhh

Reply:
```suggestion
        Usage example::
```